floris/tools/parallel_computing_interface.py

```diff
@@ -87,10 +84,8 @@ def __init__(
             object or can be an UncertaintyInterface object.
         max_workers (int): Number of parallel workers, typically equal to the number of cores
             you have on your system or HPC.
-        n_wind_direction_splits (int): Number of sectors to split the wind direction array over.
+        n_wind_condition_splits (int): Number of sectors to split the wind condition array over.
```

Comment:
In `floris.simulation`, this is generally called `findex` (see https://github.com/NREL/floris/discussions/753). For consistency, it may be best to stick with that convention.

Reply:
Updated!